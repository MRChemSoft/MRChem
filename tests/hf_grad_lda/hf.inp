world_prec = 1.0e-8                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n
world_unit = bohr

#MPI {
#  numerically_exact = true          # Guarantee identical results in MPI
#}
#
#Basis {
#  order = 6                         # Polynomial order
#}

Molecule {
$coords
H            0.000000000000     0.000000000000    -1.642850273986
F            0.000000000000     0.000000000000     0.087149726014
$end
translate = false
}

WaveFunction {
  method = LDA                      # Wave function method (HF or DFT)
}

Properties {
  geometric_derivative = true       # Compute geometric derivative
}

SCF {
  guess_type = SAD_DZ             # Type of initial guess: none, mw, gto
}
