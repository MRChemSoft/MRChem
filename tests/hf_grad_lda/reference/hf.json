{
  "input": {
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "restricted": true,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        },
        "geometric_derivative": {
          "operator": "h_nuc_grad",
          "precision": 0.001,
          "smooth_prec": 0.001
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (SVWN5)",
        "orbital_thrs": 0.01,
        "rotation": 0,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482007428676,
        -0.47659952984111875,
        0.7172401613655944
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.7020306671223611,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.36879524805203445,
            -0.11420266888268404,
            0.5863402365645545
          ],
          "vector_el": [
            23.113548846122,
            4.792870801376809,
            -7.304658181938379
          ],
          "vector_nuc": [
            -23.482344094174035,
            -4.907073470259493,
            7.890998418502933
          ]
        }
      },
      "geometric_derivative": {
        "electronic": [
          1.552213546182124,
          0.5689182828541043,
          -2.7270910932994203,
          -1.5596215672448452,
          -0.4913673996482972,
          2.4979002415089897
        ],
        "electronic_norm": 4.368458275828332,
        "nuclear": [
          -1.5689195606619417,
          -0.49421531749272585,
          2.5173387449218514,
          1.5689195606619417,
          0.49421531749272585,
          -2.5173387449218514
        ],
        "nuclear_norm": 4.25270542703273,
        "total": [
          -0.016706014479817677,
          0.07470296536137844,
          -0.20975234837756895,
          0.009297993417096473,
          0.0028479178444286712,
          -0.019438503412861685
        ],
        "total_norm": 0.22433923048823418
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -24.083540383792936,
          -1.0794529685823284,
          -0.4987562894250593,
          -0.3597972239636773,
          -0.35734334648395655
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -52.75778042449592
      },
      "scf_energy": {
        "E_ee": 55.48440786014441,
        "E_eext": 0.0,
        "E_el": -105.01594204024174,
        "E_en": -249.66778419141258,
        "E_kin": 99.30840693668151,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -99.8136298992228,
        "E_x": 0.0,
        "E_xc": -10.14097264565508
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 20,
      "routine": "mrchem.x",
      "total_cores": 20,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.63986965477407,
        "E_eext": 0.0,
        "E_el": -83.51310124051514,
        "E_en": -329.66039520385766,
        "E_kin": 166.8378423686252,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -78.3107890994962,
        "E_x": 0.0,
        "E_xc": -15.33041806005675
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 57.42489783763115,
              "E_eext": 0.0,
              "E_el": -104.47926528033932,
              "E_en": -248.66931685832276,
              "E_kin": 97.03127291258097,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.27695313932038,
              "E_x": 0.0,
              "E_xc": -10.266119172228674
            },
            "energy_total": -99.27695313932038,
            "energy_update": 20.96616403982418,
            "mo_residual": 1.2158176326129657,
            "wall_time": 3.475692876
          },
          {
            "energy_terms": {
              "E_ee": 52.78874620353378,
              "E_eext": 0.0,
              "E_el": -102.19714693067192,
              "E_en": -243.52617405024162,
              "E_kin": 98.30395656469325,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -96.99483478965298,
              "E_x": 0.0,
              "E_xc": -9.763675648657347
            },
            "energy_total": -96.99483478965298,
            "energy_update": 2.2821183496673996,
            "mo_residual": 0.40932311518783343,
            "wall_time": 3.185221082
          },
          {
            "energy_terms": {
              "E_ee": 56.80736592851206,
              "E_eext": 0.0,
              "E_el": -102.2455045130708,
              "E_en": -249.75180113048467,
              "E_kin": 100.94867928853553,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.04319237205186,
              "E_x": 0.0,
              "E_xc": -10.249748599633751
            },
            "energy_total": -97.04319237205186,
            "energy_update": 0.04835758239887866,
            "mo_residual": 0.2422112884321953,
            "wall_time": 2.802955325
          },
          {
            "energy_terms": {
              "E_ee": 55.00271919435322,
              "E_eext": 0.0,
              "E_el": -102.27310276269517,
              "E_en": -246.44561864708507,
              "E_kin": 99.19422240719922,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.07079062167624,
              "E_x": 0.0,
              "E_xc": -10.024425717162545
            },
            "energy_total": -97.07079062167624,
            "energy_update": 0.027598249624375626,
            "mo_residual": 0.10215901627898333,
            "wall_time": 3.053935892
          },
          {
            "energy_terms": {
              "E_ee": 55.419772872441705,
              "E_eext": 0.0,
              "E_el": -102.35370986633772,
              "E_en": -246.9122137820329,
              "E_kin": 99.21544283757133,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.15139772531879,
              "E_x": 0.0,
              "E_xc": -10.076711794317887
            },
            "energy_total": -97.15139772531879,
            "energy_update": 0.0806071036425493,
            "mo_residual": 0.07369410213110911,
            "wall_time": 3.232276482
          },
          {
            "energy_terms": {
              "E_ee": 55.25274626514802,
              "E_eext": 0.0,
              "E_el": -102.4142021439564,
              "E_en": -246.77202222146494,
              "E_kin": 99.16521385210167,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.21189000293747,
              "E_x": 0.0,
              "E_xc": -10.060140039741167
            },
            "energy_total": -97.21189000293747,
            "energy_update": 0.06049227761867826,
            "mo_residual": 0.019617289056692116,
            "wall_time": 3.337309571
          },
          {
            "energy_terms": {
              "E_ee": 55.29831298985835,
              "E_eext": 0.0,
              "E_el": -102.46009526729033,
              "E_en": -246.94768684922818,
              "E_kin": 99.25824217548542,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.25778312627139,
              "E_x": 0.0,
              "E_xc": -10.068963583405933
            },
            "energy_total": -97.25778312627139,
            "energy_update": 0.0458931233339257,
            "mo_residual": 0.01681669650689508,
            "wall_time": 3.441495382
          },
          {
            "energy_terms": {
              "E_ee": 55.317777896183635,
              "E_eext": 0.0,
              "E_el": -102.54591783290311,
              "E_en": -247.17480152419802,
              "E_kin": 99.38613344067474,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.34360569188418,
              "E_x": 0.0,
              "E_xc": -10.07502764556346
            },
            "energy_total": -97.34360569188418,
            "energy_update": 0.08582256561278712,
            "mo_residual": 0.02471445755442171,
            "wall_time": 3.076002689
          },
          {
            "energy_terms": {
              "E_ee": 55.32765812950924,
              "E_eext": 0.0,
              "E_el": -102.73260903345164,
              "E_en": -247.44607950373862,
              "E_kin": 99.4664938964737,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.5302968924327,
              "E_x": 0.0,
              "E_xc": -10.08068155569595
            },
            "energy_total": -97.5302968924327,
            "energy_update": 0.18669120054852328,
            "mo_residual": 0.06947978406025258,
            "wall_time": 3.083124881
          },
          {
            "energy_terms": {
              "E_ee": 55.46382607125066,
              "E_eext": 0.0,
              "E_el": -102.89157890911468,
              "E_en": -248.20282511814426,
              "E_kin": 99.95580905823594,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -97.68926676809575,
              "E_x": 0.0,
              "E_xc": -10.108388920457006
            },
            "energy_total": -97.68926676809575,
            "energy_update": 0.15896987566304688,
            "mo_residual": 0.11810960876244793,
            "wall_time": 3.246872581
          },
          {
            "energy_terms": {
              "E_ee": 55.48353719548845,
              "E_eext": 0.0,
              "E_el": -103.33198833009159,
              "E_en": -248.79495603470258,
              "E_kin": 100.10253596218881,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -98.12967618907265,
              "E_x": 0.0,
              "E_xc": -10.123105453066284
            },
            "energy_total": -98.12967618907265,
            "energy_update": 0.4404094209769056,
            "mo_residual": 0.20268531479291718,
            "wall_time": 3.298293363
          },
          {
            "energy_terms": {
              "E_ee": 55.42980248484146,
              "E_eext": 0.0,
              "E_el": -104.37067742789466,
              "E_en": -249.02027073827523,
              "E_kin": 99.34289349831576,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.16836528687573,
              "E_x": 0.0,
              "E_xc": -10.123102672776659
            },
            "energy_total": -99.16836528687573,
            "energy_update": 1.038689097803072,
            "mo_residual": 0.11378747451996371,
            "wall_time": 3.435274402
          },
          {
            "energy_terms": {
              "E_ee": 55.4844015220718,
              "E_eext": 0.0,
              "E_el": -104.88079774960188,
              "E_en": -249.51165701259285,
              "E_kin": 99.28458855419771,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.67848560858295,
              "E_x": 0.0,
              "E_xc": -10.138130813278556
            },
            "energy_total": -99.67848560858295,
            "energy_update": 0.5101203217072197,
            "mo_residual": 0.05839919967670258,
            "wall_time": 3.414853765
          },
          {
            "energy_terms": {
              "E_ee": 55.49410226822867,
              "E_eext": 0.0,
              "E_el": -104.92830529195427,
              "E_en": -249.3816421634756,
              "E_kin": 99.09496429314085,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.72599315093534,
              "E_x": 0.0,
              "E_xc": -10.135729689848189
            },
            "energy_total": -99.72599315093534,
            "energy_update": 0.04750754235239185,
            "mo_residual": 0.06632943195163668,
            "wall_time": 3.721181749
          },
          {
            "energy_terms": {
              "E_ee": 55.48440786014441,
              "E_eext": 0.0,
              "E_el": -105.01594204024174,
              "E_en": -249.66778419141258,
              "E_kin": 99.30840693668151,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.8136298992228,
              "E_x": 0.0,
              "E_xc": -10.14097264565508
            },
            "energy_total": -99.8136298992228,
            "energy_update": 0.0876367482874656,
            "mo_residual": 0.01647542372818054,
            "wall_time": 4.027545761
          }
        ],
        "wall_time": 49.832873358
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
