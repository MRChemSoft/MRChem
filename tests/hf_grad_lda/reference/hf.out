                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         mrchem-gradient                          ***
***         Git commit hash    c6b193dd4223e5895077                     ***
***         Git commit author  Roberto Di Remigio                       ***
***         Git commit date    Fri Sep 25 22:32:15 2020 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              20
 Total cores             :                                              20
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.3.1
 Git branch            : mrchem-gradient
 Git commit hash       : b79bdf06c3c82a53edb6
 Git commit author     : Roberto Di Remigio
 Git commit date       : Tue Sep 15 18:31:50 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : MPI/OpenMP

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999841722039e-01
    1    2    p          :                              9.999998855754e-01
    2    2    p          :                              9.999999997975e-01
    3    2    p          :                              9.999741700805e-01
    4    2    p          :                              9.999890982265e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.837842368625
 E-N energy              :            (au)               -329.660395203858
 Coulomb energy          :            (au)                 94.639869654774
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.330418060057
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312141019
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.513101240515
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.831078909950e+01
                         :      (kcal/mol)             -4.914076168480e+04
                         :        (kJ/mol)             -2.056049470226e+05
                         :            (eV)             -2.130945035756e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.804628057934
    1    2    p          :            (au)                  4.315085735558
    2    2    p          :            (au)                  4.740087112024
    3    2    p          :            (au)                  4.771915799504
    4    2    p          :            (au)                  5.076927750576
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.198776679454
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (SVWN5)
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          2.236068e+00         -78.310789099496         -7.831079e+01
    1          1.215818e+00         -99.276953139320         -2.096616e+01
    2          4.093231e-01         -96.994834789653          2.282118e+00
    3          2.422113e-01         -97.043192372052         -4.835758e-02
    4          1.021590e-01         -97.070790621676         -2.759825e-02
    5          7.369410e-02         -97.151397725319         -8.060710e-02
    6          1.961729e-02         -97.211890002937         -6.049228e-02
    7          1.681670e-02         -97.257783126271         -4.589312e-02
    8          2.471446e-02         -97.343605691884         -8.582257e-02
    9          6.947978e-02         -97.530296892433         -1.866912e-01
   10          1.181096e-01         -97.689266768096         -1.589699e-01
   11          2.026853e-01         -98.129676189073         -4.404094e-01
   12          1.137875e-01         -99.168365286876         -1.038689e+00
   13          5.839920e-02         -99.678485608583         -5.101203e-01
   14          6.632943e-02         -99.725993150935         -4.750754e-02
   15          1.647542e-02         -99.813629899223         -8.763675e-02
---------------------------------------------------------------------------
                      SCF converged in 15 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 99.308406936682
 E-N energy              :            (au)               -249.667784191413
 Coulomb energy          :            (au)                 55.484407860144
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.140972645655
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312141019
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -105.015942040242
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.981362989922e+01
                         :      (kcal/mol)             -6.263399789702e+04
                         :        (kJ/mol)             -2.620606473712e+05
                         :            (eV)             -2.716067116426e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -24.083540383793
    1    2    p          :            (au)                 -1.079452968582
    2    2    p          :            (au)                 -0.498756289425
    3    2    p          :            (au)                 -0.359797223964
    4    2    p          :            (au)                 -0.357343346484
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -52.757780424496
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       23.113549        4.792871       -7.304658
 Magnitude               :            (au)                       24.709629
                         :         (Debye)                       62.805607
---------------------------------------------------------------------------
 Nuclear vector          :      -23.482344       -4.907073        7.890998
 Magnitude               :            (au)                       25.254063
                         :         (Debye)                       64.189420
---------------------------------------------------------------------------
 Total vector            :       -0.368795       -0.114203        0.586340
 Magnitude               :            (au)                        0.702031
                         :         (Debye)                        1.784384
===========================================================================


===========================================================================
                           Geometric Derivative
---------------------------------------------------------------------------
 Total                   :       -0.016706        0.074703       -0.209752
                         :        0.009298        0.002848       -0.019439
 Norm                    :            (au)                        0.224339
---------------------------------------------------------------------------
 Nuclear                 :       -1.568920       -0.494215        2.517339
                         :        1.568920        0.494215       -2.517339
 Norm                    :            (au)                        4.252705
---------------------------------------------------------------------------
 Electronic              :        1.552214        0.568918       -2.727091
                         :       -1.559622       -0.491367        2.497900
 Norm                    :            (au)                        4.368458
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 53s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
